src/filters/pages.cjs

```diff
@@ -323,6 +324,24 @@ const addCallToAction = (pageURL) =>
   _.isString(pageURL) &&
   (pageURL.startsWith('/work/') || pageURL.startsWith('/services/'));
 
+/* @docs
+label: isType
+category: Filter
+note: |
+  Filters collection by a given tag,
+  expected to be one of several resource "types"
+  (types are defined in the `taxonomy.yaml` data file)
+params:
+  collection:
+    type: array
+    note: containing 11ty page objects
+  type:
+    type: type
+    note: resource type to filter by
+*/
+const isType = (collection, type) =>
+  collection.filter((page) => pageType(page.data.tags, 'tag') === type);
```

Comment:
@sanajaved7 I think this fixes the issue you and Sondra were noticing? Basically, we don't want to display all resources with a given tag -- we instead want to display all resources that are a specific canonical "type", as defined in `taxonomy.yaml`.